src/tile_source.js

```diff
@@ -8,28 +8,15 @@ import log from 'loglevel';
 export default class TileSource {
 
     constructor (source) {
+        this.name = source.name;
         this.url_template = source.url;
         // overzoom will apply for zooms higher than this
         this.max_zoom = source.max_zoom || Geo.max_zoom;
     }
 
-    // TODO fix the z adjustment for continuous zoom
```

Comment:
See comment in Tile class... not sure we should move this out of here if possible.